docs/release-notes/8.0/8.0.35-33.0.md

```diff
@@ -0,0 +1,46 @@
+# Percona XtraBackup 8.0.35-33 (2025-05-12)
+
+Get started with [Quickstart Guide for Percona XtraBackup](..//..//quickstart-overview.md).
+
+## Supported versions
+
+Starting with Percona XtraBackup 8.0.34, you can take backups of Percona Server for MySQL 8.0.34, MySQL 8.0.34, and **higher versions within the 8.0 series**.
+
+Check the latest list of all [Supported versions](..//..//supported-versions.md).
+
+## Release highlights
+
+This release provides improvements and bug fixes.
+
+We recommend that you download the Percona XtraBackup for the same platform as the MySQL-compatible server. For example, if your server is on an ARM64 platform, you should download and use the Percona XtraBackup for ARM64 for that operating system.
+
+## Improvements
+
+* [PXB-3427]: Percona XtraBackup now prepares incremental backups faster. The `--prepare` command directly applies the `.delta` files. To speed up this process, use the `--parallel=X` option, replacing `X` with the number of threads you want to use simultaneously. This option applies the delta files concurrently.
+
+* [PXB-3199]: The `xbcloud put` operations were updated to include support for [ObjectLock-enabled AWS S3 buckets].
+
+## Bug fix
+
+* [PXB-3426]: Fix xtrabackup crashes caused by Key Management Interoperability Protocol (KMIP) component keyrings usage.
+
+
+* [PXB-3490]: Fixed an invalid URL for the Boost tarball (Thanks to volver for reporting this issue.)
```

Comment:
The contribution is for PXB-3199 (in the improvements section). Thanks is for both reporting and the contribution of fix.